Lecture2/Task1/builder/Guitar.java

```diff
@@ -0,0 +1,52 @@
+package builder;
+
+import product.Brand;
+
+public class Guitar {
+	private Brand brand;
+	private String color;
+	private int stringsCount;
+
+	public Guitar() {
+		this.brand = brand;
```

Comment:
brand is null 
this expression doesn't have any sense
you should pass all required param in constructor
public Guitar(Brand brand, String color) {
this.brand = brand
etc
}

Reply:
fixed